azure-pipelines.yml

```diff
@@ -130,13 +130,6 @@ jobs:
     failOnStderr: true
     displayName: 'Upgrade pip and setuptools'
 
-  - bash: |
-      pycairo_url=$(python .azure-ci/python-igraph_dependencies_win.py '$(python_ver)' 'pycairo')
-      igraph_url=$(python .azure-ci/python-igraph_dependencies_win.py '$(python_ver)' 'python-igraph')
-      pip install "$pycairo_url" "$igraph_url"
-    failOnStderr: true
-    displayName: 'Install pycairo and python-igraph'
-
```

Comment:
Just to confirm, python-igraph is now installed as a normal dependency?

Reply:
That's precisely the point of the PR, yes :)